packages/salesforcedx-test-utils-vscode/src/testrunner.ts

```diff
@@ -30,29 +30,34 @@ let mocha = new Mocha({
   reporter: 'mocha-multi-reporters'
 });
 
-function configure(mochaOpts: any): void {
+function configure(mochaOpts: any, xmlOutputDirectory: string): void {
   if (mochaOpts.reporter == null) {
     // default to 'mocha-multi-reporters' (to get xunit.xml result)
     mochaOpts.reporter = 'mocha-multi-reporters';
   }
   if (!mochaOpts.reporterOptions) {
     let xmlPath = '';
-    // There were some oddities on Windows where the mocha execution would be inside the downloaded version of vscode and store the test result file there
-    // This will fix the pathing for windows. This behavior is not seen in the system-tests, appears to be only when we use vscode's test bin to run tests
-    if (process.platform === 'win32') {
-      xmlPath = paths.normalize(paths.join(process.cwd(), '..', '..'));
-    }
+
+    const junitOutputFileLocation = paths.join(
+      xmlOutputDirectory,
+      'junit-custom-vscodeIntegrationTests.xml'
+    );
+    const xunitOutputFileLocation = paths.join(
+      xmlOutputDirectory,
+      'xunit-vscodeIntegrationTests.xml'
+    );
+
+    console.log('Output locations for reporters: ', {
+      junitOutputFileLocation,
+      xunitOutputFileLocation
+    });
```

Comment:
Log it out so this is less timely in figuring out next time we run into a related issue.